### ISSUE TITLE
Archived confirmation message blank

### ISSUE THREAD

**Issue by maxime-rainville** [silverstripe/silverstripe-versioned-admin#94]
# Affected version
SS4.3

# Description

When archiving a DataObject through a gridfield, the message is blank. Delete works fine.
![image](https://user-images.githubusercontent.com/1168676/51003765-e86c3d80-159c-11e9-9681-19bb31021ee4.png)

I'm pretty sure I had a look at this before the 4.3 release, but I must have mess something up.

Identified through https://github.com/silverstripe/silverstripe-versioned/issues/200

# Steps to reproduce

* Access a ModelAdmin managing a versioned DataObject.
* Try archiving an record.

**Comment by robbieaverill** [comment]
I don't think we've updated translations in a while, which is why `Admin.ARCHIVECONFIRMMESSAGE` isn't in `client/lang/en.js`, so the fallback string is being used. The implementation doesn't have a fallback message, which is why it's blank. PR to fix this at https://github.com/silverstripe/silverstripe-admin/pull/842

I've also pushed the translations to Transifex, which will ensure that they're picked up the next time we _pull_ from Transifex in the master en.js file.
